enterprise/frontend/src/metabase-enterprise/metabot/state/reducer.ts

```diff
@@ -9,20 +10,22 @@ import type {
 import { sendMessageRequest } from "./actions";
 
 export interface MetabotState {
-  isProcessing: boolean;
-  userMessages: string[];
   confirmationOptions: Record<string, MetabotReaction[]> | undefined;
+  isProcessing: boolean;
   lastSentContext: MetabotChatContext | undefined;
   lastHistoryValue: MetabotHistory | undefined;
+  sessionId: string | undefined;
+  userMessages: string[];
   visible: boolean;
 }
```

Comment:
maybe this can find a better home?

Reply:
Where the Metabot session id lives or the state type definition?

Reply:
the state type

Reply:
I have it on my task list to rework the enterprise state, so I'll look at moving it as part of that change.